spec/rails_helper.rb

```diff
@@ -54,7 +54,7 @@
 
   config.before do
     allow(SchoolGroupJobsFeature).to receive(:enabled?).and_return(true)
-    allow(MultiSchoolJobsFeature).to receive(:enabled?).and_return(false)
+    allow(MultiSchoolJobsFeature).to receive(:enabled?).and_return(true)
```

Comment:
Do we still need these 2 lines?

Reply:
Until the feature flags are removed I think so